core/common/blob.hpp

```diff
@@ -6,13 +6,21 @@
 #ifndef KAGOME_BLOB_HPP
 #define KAGOME_BLOB_HPP
 
+#include <array>
+
 #include <boost/format.hpp>
 
 #include "common/hexutil.hpp"
-#include "common/result.hpp"
 
 namespace kagome::common {
 
+  /**
+   * Error codes for exceptions that may occur during blob initialization
+   */
+  enum class BlobError {
+    kIncorrectLength = 1
```

Comment:
During the last meeting, we decided that enum entries should be `SCREAMING_CASE`. Please, rename all new enums according to this rule